pingvin-share/docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pingvin-share_web_1
+      APP_PORT: 6453
+      PROXY_AUTH_ADD: "false"
+  
+  web:
+    image: stonith404/pingvin-share:v0.27.0@sha256:ebcd050ddcf0fc6e2f14944e66f478825697479cb79feaec7d989fa4a45a0635
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    ports:
+      - 6453:3000
+    volumes:
+      - "${APP_DATA_DIR}/data:/opt/app/backend/data"
+      - "${APP_DATA_DIR}/data/images:/opt/app/frontend/public/img"
```

Due to privilege issues on older umbrelOS devices, we need to add those directories to the app. Please create the following files:
- `pingvin-share/data/images/.gitkeep`
- `pingvin-share/data/backend/.gitkeep`

The problem is that on older versions of umbrelOS (again backwards compatibility) the permissions of automatically created repositories are wrong. By supplying a .gitkeep file, those directories are getting created by installing the app automatically.
Also maybe it would be better to save the backend data separately to prevent the clashing of files:
```suggestion
      - "${APP_DATA_DIR}/data/backend:/opt/app/backend/data"
      - "${APP_DATA_DIR}/data/images:/opt/app/frontend/public/img"
```